### PR TITLE
Changing brand page style

### LINKED ISSUE #14398: FO Brands page style is broken
**Describe the bug**
From the BO, a merchant can enable a "All brands" page that shows all brands (a.k.a manufacturers) on the FO.

The page style is broken (see screenshot)

**To Reproduce**
1. Go to FO on `http://localhost:8888/brands` or `http://localhost:8888/en/brands` (depends if you have 1 or multiple languages enabled)
2. See broken display

**Screenshots**
Capture d’écran 2019-06-27 à 16 00 14

**Additionnal information**
PrestaShop version: 1.7.5.x, 1.7.6.x, develop

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Brands page had not any style
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #14398 
| How to test?  | Just go to this page : http://localhost/prestashop/index.php?controller=manufacturer or /brands

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16857)

### CODE REVIEW

File: themes/classic/_dev/css/components/brands.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#manufacturer {
+    #main {
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/brands.scss]: Use 2 spaces instead.

File: themes/classic/_dev/css/components/brands.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            flex-wrap: wrap;
+
+            .brand {
+                padding: 10px;
+                margin: 5px;
+                text-align: center;
+                background: white;
+                transition: .4s ease-out;
+                width: calc(20% - 10px);
+                text-align: center;
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/brands.scss]: Duplicate
```suggestion
```

File: themes/classic/_dev/css/components/brands.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                &-img {
+                    width: 200px;
+                    margin: auto;
+                }
+            }
+        }
+    }
+}
+
+@media screen and (max-width: 768px) {
```
Comment (PierreRambaud) [inline themes/classic/_dev/css/components/brands.scss]: could we use the boostrap variables for 768px? https://getbootstrap.com/docs/4.0/layout/grid/#sass-mixins

**Review by PierreRambaud — APPROVED** [review]

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**